lstmcpipe/stages/mc_process_dl1.py

```diff
@@ -169,16 +170,16 @@ def r0_to_dl1(
         Ratio of training data. Default = 0.5
     rng : int
         Random seed for random processes. Default = 42
-    n_r0_per_dl1_job : int
+    dl1_files_per_job : int
         Number of r0 files processed by each r0_to_dl1 batched stage. If set to None (Default), see below the `usual
         production` case.n_r0_files_per_dl1_job
 
         If the number of r0 files found in `input_dir` is less than 100, it is consider to be a test on a small
-        production. Therefore, the number of r0 files treated per batched stage will be set to 10.
+        production. Therefore, the number of r0 files treated per batched stage (r0_to_dl1 or dl1ab) will be set to 10.
 
-        Usual productions have =>1000 r0 files, in this case, the number of files per job will be fixed to 50 (in case
-        of gamma and electrons), and 100 for protons. Because of this protons end up in the long queue and other particles
-        are submitted to the short queue.
+        Usual productions have =>1000 r0 files, in this case, the number of files per job (r0_to_dl1 or dl1ab) will be
+        fixed to 50 (in case of gamma and electrons), and 100 for protons. Because of this protons end up in the
+        long queue and other particles are submitted to the short queue.
```

Comment:
Actually we dont differentiate anymore, see below

Reply:
I think there are two different things.

It is not the same the number of batched 'r0_to_dl1' jobs, that the number of files that each of this jobs will sub-process.

n_of_parallel_jobs would determine the array length for a particle, i.e., 20, then, no more than 20 job-particle would be able to run at the same time on the cluster. However, we could have created 100 file lists, that will imply 100 particle jobs to be run.